server/chart_config/climate.json

```diff
@@ -78,5 +78,17 @@
       "scale": false
     },
     "isChoropleth": true
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-Annual_emissions_greenhouse_gas",
+    "title": "Annual greenhouse gas emissions",
```

Comment:
We should clarify that these are large facility emissions:

"Annual greenhouse gas emissions from large facilities"

Reply:
Fixed in https://github.com/datacommonsorg/website/pull/1157